collector/log4j-socket/pom.xml

```diff
@@ -93,25 +93,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
-                <groupId>org.apache.felix</groupId>
-                <artifactId>maven-bundle-plugin</artifactId>
-                <configuration>
-                    <instructions>
-                        <Import-Package>
-                            !com.ibm.uvm.tools,
-                            *
-                        </Import-Package>
-                        <Private-Package>
-                            org.apache.karaf.decanter.collector.log.socket,
```

Comment:
Instead of removing completely the instructions, I would add `org.apache.karaf.decanter.collector.utils` as private package (as in the other collectors).

Reply:
@jbonofre That is actually still in the pom:

    <Private-Package>
                            org.apache.karaf.decanter.collector.log.socket,
                            org.apache.karaf.decanter.collector.utils
                        </Private-Package>

The problem is that there were two maven-bundle-plugin definitions.

Reply:
Good catch, thanks !